canonical/snapcraft.io
# Add banner image upload

## Done
Added a component for banner image uploads

## QA
- Go to https://snapcraft-io-4078.demos.haus/<snap-name>/listing?show_react_listing=true
- Check that you can upload and remove banner images

## Issue
Fixes https://github.com/canonical/marketplace-tribe/issues/2644
